src/main/java/com/fasterxml/uuid/UUIDType.java

```diff
@@ -13,8 +13,10 @@ public enum UUIDType {
     NAME_BASED_MD5(3),
     RANDOM_BASED(4),
     NAME_BASED_SHA1(5),
-    UNKNOWN(0)
-    ;
+    DB_LOCALITY(6),
```

Comment:
naming?

Reply:
Sounds good in general, I should probably read the spec but sounds reasonable.